### PR TITLE
DBZ-3317 Apply common format of decimal mode doc to Oracle

### PR DESCRIPTION
**Pull request by roldanbob**
[DBZ-3317](https://issues.redhat.com/browse/DBZ-3317)

Reworks Decimal types section in Oracle connector doc to reflect the way that the same information is presented for other connectors.

Tested in local Antora  and downstream builds.

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 1]
```diff
@@ -1332,15 +1332,92 @@ boolean.selector=.*MYTABLE.FLAG,.*.IS_ARCHIVED
...
+
+|`SMALLMONEY`
+|`BYTES`
+a|`org.apache.kafka.connect.data.Decimal` +
+The `scale` schema parameter contains an integer that represents how many digits the decimal point shifted.
+
+|`MONEY`
+|`BYTES`
+a|`org.apache.kafka.connect.data.Decimal` +
+The `scale` schema parameter contains an integer that represents how many digits the decimal point shifted.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:1367]: These types are not supported with Oracle AFAIK // CC @Naros.

**Review by gunnarmorling — COMMENTED** [review]
LGTM, only the monetary types shouldn't be mentioned, unless @Naros can confirm they actually are supported with Oracle.

File: documentation/modules/ROOT/pages/connectors/oracle.adoc (hunk 1 above)
Comment (Naros) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1367]: Hi guys, so `SMALLMONEY` and `MONEY` are not valid data types for Oracle.  In fact these are represented most often using other data types such as `DECIMAL`, `FLOAT`, `NUMERIC`, `NUMBER` with appropriate precision and scale values to fit the desired outcome.  

Furthermore, we already include these data types in the Numeric Types section of the docs.  It doesn't make much sense to also list these in the Decimal Types section too.  Is there a logical way we can embed the implied behavior of the `decimal.handling.mode` in the notes section for the respective data types or add a footnote?

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1367]: @Naros Sorry, but I'm a little bit lost here. I'm not sure what you're asking. I get that we should remove the `*MONEY` types, and that those types are typically represented via other data types  that the _Numeric Types_ section already mentions. 
But I'm not clear on what you mean by 
_Is there a logical way we can embed the implied behavior of the `decimal.handling.mode` in the notes section for the respective data types or add a footnote_

Might be good to have a quick call to clarify.

Comment (Naros) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1367]: What I mean @roldanbob is that we have `NUMERIC` and `DECIMAL` data types listed under Numeric Types and then again under Decimal Types but with varied information based on the `decimal.handling.mode` setting.  I don't think we should be duplicating was my point.  Can we represent this in a way that doesn't involve duplication?

Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1367]: Ah yes, good point; each type mapping should be listed only once. Good idea to see how it's done for the other connectors; perhaps make "decimal types" and the different mappings depending on the decimal.handling.mode option a sub-section of "numeric types", and discuss the decimal types only in that sub-section?

**Comment by gunnarmorling** [comment]
@roldanbob, where are we with this one? Anything missing (apart from the required rebase for resolving the merge conflict)?

**Comment by gunnarmorling** [comment]
Ok, please let me know if you need any further help to clarify things.

**Pull request closed without merging**